### ISSUE TITLE
Notification - WCAG rule 1.3.6

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/grommet#7603]
**Failed rule**

1.3.6
Identify Purpose
Level AAA

In content implemented using markup languages, the purpose of user interface components, icons, and regions can be programmatically determined.

---

**Notes**

This is a Level AAA fail

Close button should be more descriptive (needs aria-label)
Role status could be used for a toast notification but not for an inline or global notification.

**Comment by jcfilben** [comment]
Marking as enhancement since this is a AAA rule
